lib/index.js

```diff
@@ -12,13 +8,13 @@ function constructResponse (server) {
     argv: Hoek.clone(process.argv),
     execArgv: Hoek.clone(process.execArgv),
     cwd: process.cwd(),
-    // This should probably be a clone(), but Hoek is currently breaking
-    // when working with process.env on 0.10
-    env: process.env
+    env: Hoek.clone(process.env)
```

Comment:
Yes. Travis tests pass so, we're good.

https://github.com/hapijs/hoek/issues/162